data/blog/discord-hacktoberfest-2020.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Community Discord Server and Hacktoberfest
+author:
+- erickzhao
+date: '2020-10-01'
+---
+
+Join us for community bonding and a month-long celebration of open-source.
+
+---
+
+![Hacktoberfest and Discord banner](https://user-images.githubusercontent.com/16010076/94834005-add7b380-03c4-11eb-8dfc-af5e3972fa53.png)
+
+
+# Electron Community Discord Launch
+Electron’s [Outreach Working Group](https://github.com/electron/governance/tree/master/wg-outreach) is excited to
+announce the launch of our official community Discord server!
+
+## Why a new Discord server?
+In its early days as the backbone of the [Atom text editor](https://atom.io/), community discussion on the Electron
+framework occurred in a single channel in Atom’s Slack workspace. As time passed and the two projects were increasingly
+decoupled, the relevance of the Atom workspace to the Electron project decreased, and maintainer participation in the
+Slack channel declined in the same manner.
+
+Up until now, we had still been redirecting our broader community to the Atom Slack workspace, even though we’ve
+had many reports from folks who have had trouble receiving invitations, and few of our core maintainers were
+frequenting the channel.
+
+We’re setting up this shiny new server to be a central discussion hub for the community where you can get the latest
+news on all things Electron.
+
+## Get in here!
+So far, the server’s membership consists of a few maintainers who have been working together to set it up, but we’re
+so excited to chat with you all! Come ask for help, keep up to date with Electron releases, or just hang out with other
+developers. We’ve got a handy [invite for you](https://discord.gg/H6uTh7m) that’ll give you access to the server!
+
+# Hacktoberfest 2020
+As a large and long-running open-source project, Electron wouldn’t have been nearly as successful without all the
+contributions from its community, from code submissions to bug reports to documentation changes, and much more.
+That’s why we believe in the importance of participating in Hacktoberfest to usher in a wider community of developers
+of all skill levels into the project.
+
+## Odds and ends
+This year, we don’t have a wider project to give you all to work on, but we’d like to focus on opportunities to contribute
+across the Electron JavaScript ecosystem.
+
+Look out for issues tagged `hacktoberfest` across our various repositories, including the main [electron/electron](https://github.com/electron/electron)
```

Comment:
This seems more optimal since shows all issues tagged by `hacktoberfest` label in electron org

```suggestion
Look out for issues tagged `[hacktoberfest](https://github.com/search?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest+org%3Aelectron)` across our various repositories, including the main [electron/electron](https://github.com/electron/electron)
```

Reply:
We'd still need the separate link for Electron Forge.

Reply:
what about just add `org:electron-userland` to the search query, this also adds `electron-forge`

Reply:
[Seems to work](https://github.com/search?q=is%3Aissue+is%3Aopen+label%3Ahacktoberfest+org%3Aelectron+org%3Aelectron-userland)